server/tests/end_to_end_decentralized.ts

```diff
@@ -82,18 +80,18 @@ describe('end to end decentralized', function () {
     with other ready peers. The input will vary with model architecture and training data. If secure is true,
     the client will implement secure aggregation. If it is false, it will be a clear text client.
      */
-  async function makeClient (input: number[][], secure: boolean): Promise<Weights> {
+  async function makeClient (input: number[], secure: boolean): Promise<WeightsContainer> {
```

Comment:
(this is from older code, not related to current PR much. why is input a number type and not a model already?)

Reply:
This file is a part of tests. The input here was a list of weights simplified as `number[][]`(<-> `Weights[]` <-> `tf.Tensor[][]`) since we didn't have any utilities to get a `Weights` object from numbers easily, which is convenient for tests. Now, `WeightsContainer` accepts raw numbers as input so we can directly pass those.

I left the `input[]` in `makeClient` to avoid repeatedly creating `WeightsContainer`s and leaving it to the function.